tools/pipelines/templates/build-npm-package.yml

```diff
@@ -259,6 +259,15 @@ stages:
           # and Istanbul's cobertura reporter doesn't escape them causing parse error when we publish
           # A quick fix to patch the file with sed. (See https://github.com/bcoe/c8/issues/302)
           - ${{ if eq(variables['testCoverage'], true) }}:
+            - task: Npm@1
+              displayName: npm run ci:test:coverage
+              inputs:
+                command: 'custom'
+                workingDir: ${{ parameters.buildDirectory }}
+                customCommand: 'run ci:test:coverage'
+            # Some webpacked file using externals introduce file name with quotes in them
+            # and Istanbul's cobertura reporter doesn't escape them causing parse error when we publish
+            # A quick fix to patch the file with sed. (See https://github.com/bcoe/c8/issues/302)
```

Comment:
Remove duplicated comment. Either leave at top or move next to the task where sed is used.

Reply:
> Remove duplicated comment. Either leave at top or move next to the task where sed is used.

I do not why the duplicated comment exists after I checked the commit history. I just removed one